### PR TITLE
OP-780 - Feature/op 780 oh core make spring use xxx ACT

### PR DESCRIPTION
**Pull request by goto-eof**
I forgot to push also the gui project changes. Infact if my previous OP-780 PR will be merged, gui will fail to start.

### CODE REVIEW

File: rsc/settings.properties.dist [hunk 1]
```diff
@@ -43,4 +43,4 @@ ALLOWMULTIPLEOPENEDBILL=yes
 PATIENTBILLGROUPED=PatientBillGrouped
 PATIENTBILLSTATEMENT=PatientBillStatement
 DEBUG=no
-
+PATIENTPHOTO=DB
```
Comment (mwithi) [inline rsc/settings.properties.dist:46]: this change belongs to OP-879

File: bundle/language_en.properties [hunk 2]
```diff
@@ -1176,6 +1176,8 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = Parents are together
 angal.patient.patientbrowser.title                                                                     = Patient Browser
 angal.patient.patientphoto                                                                             = Patient Photo
+angal.patient.patientphoto.error.title                                                                 = Patient Photo Error
+angal.patient.patientphoto.error.filenotfound.msg                                                      = Patient photo file was not found
```
Comment (mwithi) [inline bundle/language_en.properties:1180]: this change belongs to [OP-879](https://openhospital.atlassian.net/browse/OP-879).

All other langues should not be modified

File: src/main/java/org/isf/admission/gui/AdmissionBrowser.java [hunk 3]
```diff
@@ -1865,7 +1865,7 @@ private JButton getSaveButton() {
 					admission.setType("N");
 				}
 
-				admission.setDeleted("N");
+				admission.setActive(1);
```
Comment (mwithi) [inline src/main/java/org/isf/admission/gui/AdmissionBrowser.java:1868]: I think we can just remove this set

File: bundle/language_ar.properties [hunk 4]
```diff
@@ -314,6 +314,8 @@ angal.patient.nextkin                                                       = ذ
 angal.patient.note                                                          = ملاحظة
 angal.patient.parenttogether                                                = الآباء معا
 angal.patient.patientphoto                                                  = صورة المريض
+angal.patient.patientphoto.error.title                                      = 
+angal.patient.patientphoto.error.filenotfound                               =
```
Comment (mwithi) [inline bundle/language_ar.properties:318]: Please remove empty lines

File: bundle/language_es.properties [hunk 5]
```diff
@@ -1175,6 +1175,8 @@ angal.patient.parenttogether
 angal.patient.patientbrowser.title                                                                     = Lista Pacientes
 angal.patient.patientphoto                                                                             = Foto Paciente
 angal.patient.patientselection.title                                                                   = Seleccionar Paciente
+angal.patient.patientphoto.error.title                                                                 = Error en la foto del paciente
+angal.patient.patientphoto.error.filenotfound                                                          = No se encontr� la foto del paciente.
```
Comment (mwithi) [inline bundle/language_es.properties:1179]: There's no need to provide translations as they will come from Transifex once merged on develop, thanks

File: bundle/language_fr.properties [hunk 6]
```diff
@@ -1174,6 +1174,8 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = Parents vivant ensemble
 angal.patient.patientbrowser.title                                                                     = Liste Patients
 angal.patient.patientphoto                                                                             = Photo du Patient
+angal.patient.patientphoto.error.title                                                                 = Erreur de photo du patient
+angal.patient.patientphoto.error.filenotfound                                                          = La photo du patient n'a pas �t� trouv�e.
```
Comment (mwithi) [inline bundle/language_fr.properties:1178]: There's no need to provide translations as they will come from Transifex once merged on develop, thanks

File: bundle/language_it.properties [hunk 7]
```diff
@@ -1175,7 +1175,9 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = I genitori sono insieme
 angal.patient.patientbrowser.title                                                                     = Elenco Pazienti
 angal.patient.patientphoto                                                                             = Foto
-angal.patient.patientselection.title                                                                   = Selezione Paziente
+angal.patient.patientphoto.error.title                                                                 = Patient Photo Error
+angal.patient.patientphoto.error.filenotfound                                                          = Errore della foto del paziente
+angal.patient.patientselection.title                                                                   = Non � stata trovata la foto del paziente.
```
Comment (mwithi) [inline bundle/language_it.properties:1180]: There's no need to provide translations as they will come from Transifex once merged on develop, thanks

File: bundle/language_pt.properties [hunk 8]
```diff
@@ -410,6 +410,8 @@ angal.patient.nextkin                                                       = Pa
 angal.patient.note                                                          = Nota
 angal.patient.parenttogether                                                = Pais juntos
 angal.patient.patientphoto                                                  = Foto de paciente
+angal.patient.patientphoto.error.title                                      = Erro de foto do paciente
+angal.patient.patientphoto.error.filenotfound                               = A foto do paciente n�o foi encontrada.
```
Comment (mwithi) [inline bundle/language_pt.properties:414]: There's no need to provide translations as they will come from Transifex once merged on develop, thanks

File: bundle/language_sw.properties [hunk 9]
```diff
@@ -352,6 +352,8 @@ angal.patient.nextkin                                                       = Mr
 angal.patient.note                                                          = Kidokezo
 angal.patient.parenttogether                                                = Wazazi pamoja
 angal.patient.patientphoto                                                  = Picha ya Mgonjwa
+angal.patient.patientphoto.error.title                                      = Patientfotofel
+angal.patient.patientphoto.error.filenotfound                               = Patientfoto hittades inte.
```
Comment (mwithi) [inline bundle/language_sw.properties:356]: There's no need to provide translations as they will come from Transifex once merged on develop, thanks

File: bundle/language_it.properties [hunk 10]
```diff
@@ -1175,7 +1175,7 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = I genitori sono insieme
 angal.patient.patientbrowser.title                                                                     = Elenco Pazienti
 angal.patient.patientphoto                                                                             = Foto
-angal.patient.patientselection.title                                                                   = Selezione Paziente
+angal.patient.patientselection.title                                                                   = Non � stata trovata la foto del paziente.
```
Comment (mwithi) [inline bundle/language_it.properties:1178]: Please revert this change

File: src/main/java/org/isf/menu/gui/Login.java [hunk 11]
```diff
@@ -125,7 +125,7 @@ public void keyReleased(KeyEvent event) {
 	public Login(MainMenu parent) {
 		super(parent, MessageBundle.getMessage("angal.login.title"), true);
 
-		usersListLogin = GeneralData.getGeneralData().getUSERSLISTLOGIN();
+		//usersListLogin = GeneralData.getGeneralData().getUSERSLISTLOGIN();
```
Comment (mwithi) [inline src/main/java/org/isf/menu/gui/Login.java:128]: Why this instruction has been commented out?

Comment (AndreiDodu) [reply src/main/java/org/isf/menu/gui/Login.java:128]: Ignore it.

Comment (mwithi) [inline src/main/java/org/isf/menu/gui/Login.java:128]:
```suggestion
		usersListLogin = GeneralData.getGeneralData().getUSERSLISTLOGIN();
```

File: bundle/language_ar.properties (hunk 4 above)
Comment (mwithi) [inline bundle/language_ar.properties:318]:
```suggestion
```

File: bundle/language_fr.properties (hunk 6 above)
Comment (mwithi) [inline bundle/language_fr.properties:1178]:
```suggestion
```

File: bundle/language_it.properties (hunk 10 above)
Comment (mwithi) [inline bundle/language_it.properties:1178]:
```suggestion
angal.patient.patientselection.title                                                                   = Selezione Paziente
```

**Comment by mwithi** [comment]
it seems that language files for fr and it are corrupted: need to replace with develop version and push them again.
